eng/targets/RepoToolset/ProjectDefaults.props

```diff
@@ -37,13 +37,10 @@
        - Embed PDBs to make it easier to debug Jenkins crash dumps.
 
       Developer build:
-       - Build Windows PDBs in order to provide stack traces in xUnit on .NET Framework (4.7.1)
-       - TODO: https://github.com/dotnet/roslyn/issues/25184
```

Comment:
📝 This pull request should mention this issue

❔  Have you verified that xUnit now works in Test Explorer with this?